Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -65,20 +76,37 @@ public class RoutingTable
         /// <summary>
         /// The number of peers in the table.
         /// </summary>
-        public int Count => _buckets.Sum(bucket => bucket.Count);
+        public int Count => _buckets.Sum(bucket => bucket.Count) + _staticPeerStates.Length;
+
+        /// <summary>
+        /// An <see cref="IEnumerable{T}"/> of static peers in the table.
+        /// </summary>
+        public IEnumerable<BoundPeer> StaticPeers => _staticPeerStates
+            .Select(state => state.Peer)
+            .ToArray();
 
         /// <summary>
         /// An <see cref="IEnumerable{T}"/> of peers in the table.
         /// </summary>
-        public IEnumerable<BoundPeer> Peers => NonEmptyBuckets
-            .SelectMany((bucket, _) => bucket.Peers)
-            .ToList();
+        public IEnumerable<BoundPeer> Peers => StaticPeers
+            .Union(NonEmptyBuckets.SelectMany(bucket => bucket.Peers))
+            .ToArray();
+
+        /// <summary>
+        /// An <see cref="IEnumerable{T}"/> of <see cref="PeerState"/> of static peers in the table.
+        /// </summary>
+        public IEnumerable<PeerState> StaticPeerStates => _staticPeerStates.ToArray();
 
         /// <summary>
         /// An <see cref="IEnumerable{T}"/> of <see cref="PeerState"/> of peers in the table.
         /// </summary>
-        public IEnumerable<PeerState> PeerStates =>
-            NonEmptyBuckets.SelectMany(bucket => bucket.PeerStates);
+        public IEnumerable<PeerState> PeerStates => StaticPeerStates
```

Comment:
How about typing this as `IReadOnlyList<PeerState>` instead of `IEnumerable<PeerState>` to get rid of [possible multiple enumeration warning][1]?

[1]: https://www.jetbrains.com/help/resharper/PossibleMultipleEnumeration.html

```suggestion
        public IReadOnlyList<PeerState> PeerStates => StaticPeerStates
```